docs/tutorials/3DTiles-mesh-b3dm.md

```diff
@@ -0,0 +1,279 @@
+
+```js
+    // Define crs projection that we will use (taken from https://epsg.io/3946, Proj4js section)
+    itowns.proj4.defs('EPSG:3946', 
+    '+proj=lcc +lat_1=45.25 +lat_2=46.75 +lat_0=46 +lon_0=3 +x_0=1700000 +y_0=5200000 +ellps=GRS80' + 
+    '+towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+
+    // Define geographic extent: CRS, min/max X, min/max Y
+    var extent = new itowns.Extent( 'EPSG:3946',
+        1837816.94334, 1847692.32501,
+        5170036.4587, 5178412.82698);
+
+    // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+    var viewerDiv = document.getElementById('viewerDiv');
+
+    // Instanciate PlanarView*
+    var cameraCoord = new itowns.Coordinates('EPSG:3946', 1841980,
+        5175682, 3000)
+    var view = new itowns.PlanarView(viewerDiv, extent, { placement: {
+        coord: cameraCoord, heading: 30, range: 4000, tilt: 30 } });
+
+    // Add a WMS imagery source
+    var wmsImagerySource = new itowns.WMSSource({
+        extent: extent,
+        name: 'Ortho2009_vue_ensemble_16cm_CC46',
+        url: 'https://download.data.grandlyon.com/wms/grandlyon',
+        version: '1.3.0',
+        crs: 'EPSG:3946',
+        format: 'image/jpeg',
+    });
+
+    // Add a WMS imagery layer
+    var wmsImageryLayer = new itowns.ColorLayer('wms_imagery', {
+        updateStrategy: {
+            type: itowns.STRATEGY_DICHOTOMY,
+            options: {},
+        },
+        source: wmsImagerySource,
+    });
+
+    view.addLayer(wmsImageryLayer);
+
+    // Add a WMS elevation source
+    var wmsElevationSource = new itowns.WMSSource({
+        extent: extent,
+        url: 'https://download.data.grandlyon.com/wms/grandlyon',
+        name: 'MNT2012_Altitude_10m_CC46',
+        crs: 'EPSG:3946',
+        width: 256,
+        format: 'image/jpeg',
+    });
+
+    // Add a WMS elevation layer
+    var wmsElevationLayer = new itowns.ElevationLayer('wms_elevation', {
+        useColorTextureElevation: true,
+        colorTextureElevationMinZ: 144,
+        colorTextureElevationMaxZ: 622,
+        source: wmsElevationSource,
+    });
+
+    view.addLayer(wmsElevationLayer);
+```
+
+## Adding the 3D Tiles Layer
+
+The 3D Tiles dataset we are using in this example has a geometry that is compressed with 
+[Draco compression](https://google.github.io/draco/) for better performances. Therefore, we need to enable the draco 
+decoder:
+
+```js
+itowns.enableDracoLoader('./libs/draco/');
+```
+
+As for every data displayed in iTowns, we first need to define a data `Source`.
+Our data is in the 3d-tiles format, so we can use iTowns `{@link C3DTilesSource}` : 
+
+```js
+const buildingsSource = new itowns.C3DTilesSource({
+    url: 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/3dtiles-dataset/' +
```

Comment:
The branch `3dtiles-dataset` in the url needs to be updated to `master`.

Reply:
well spotted, done